src/services/programService/httpClient.js

```diff
@@ -27,6 +27,13 @@ import fetch from 'node-fetch';
 import { PROGRAM_SERVICE_HTTP_ROOT } from '../../config';
 import { restErrorResponseHandler } from '../../utils/restUtils';
 
+//get DataCenter by Program short name handler
+const getDataCenterByShortName = (shortName, dataCenterResponse) => {
+	return dataCenterResponse.filter((dataCenterObject) => {
+		return dataCenterObject.shortName === shortName;
+	});
+};
+
```

Comment:
write a function that filters the endpoint response by program shortName. If shortName is used, the result of this function is returned to the resolver instead of the entire endpoint response.